packages/@react-spectrum/s2/src/Breadcrumbs.tsx

```diff
@@ -71,43 +91,123 @@ const wrapper = style<BreadcrumbsStyleProps>({
   }
 }, getAllowedOverrides());
 
+const InternalBreadcrumbsContext = createContext<BreadcrumbsProps<any>>({});
+
 function Breadcrumbs<T extends object>(props: BreadcrumbsProps<T>, ref: DOMRef<HTMLOListElement>) {
   [props, ref] = useSpectrumContextProps(props, ref, BreadcrumbsContext);
+  let domRef = useDOMRef(ref);
   let {
     UNSAFE_className = '',
     UNSAFE_style,
     styles,
     size = 'M',
     children,
+    isDisabled,
     ...otherProps
   } = props;
-  let domRef = useDOMRef(ref);
+
   return (
-    <RACBreadcrumbs
-      {...otherProps}
-      ref={domRef}
-      style={UNSAFE_style}
-      className={UNSAFE_className + wrapper({
-        size
-      }, styles)}>
-      <BreadcrumbsContext.Provider value={props}>
-        {children}
-      </BreadcrumbsContext.Provider>
-    </RACBreadcrumbs>
+    <Provider
+      values={[
+        [InternalBreadcrumbsContext, {size, isDisabled}]
+      ]}>
+      <CollapsingCollection containerRef={domRef} onAction={props.onAction}>
+        <RACBreadcrumbs
+          {...otherProps}
+          isDisabled={isDisabled}
+          ref={domRef}
+          style={UNSAFE_style}
+          className={UNSAFE_className + wrapper({
+            size
+          }, styles)}>
+          {children}
+        </RACBreadcrumbs>
+      </CollapsingCollection>
+    </Provider>
   );
 }
 
+let BreadcrumbMenu = (props: {items: Array<Node<any>>, onAction: BreadcrumbsProps<unknown>['onAction']}) => {
+  let {items, onAction} = props;
+  let {direction} = useLocale();
+  let {size, isDisabled} = useContext(InternalBreadcrumbsContext);
+  // TODO localize See more
+  // somehow remove the "link out" icon in Menus?
+  return (
+    <UNSTABLE_CollectionRendererContext.Provider value={UNSTABLE_DefaultCollectionRenderer}>
+      <li className={breadcrumbStyles({size, isDisabled, isMenu: true})}>
+        <MenuTrigger>
+          <S2ActionButton isDisabled={isDisabled} isQuiet aria-label="See more"><FolderIcon /></S2ActionButton>
+          {/* @ts-ignore - how are these onAction's not compatible? */}
+          <Menu items={items} onAction={onAction}>
+            {(item: Node<any>) => (
+              <MenuItem
+                {...item.props.originalProps}
+                key={item.key}>
+                <Text slot="label">
+                  {item.props.children({size, isInCtx: true, isCurrent: false, isMenu: true})}
+                </Text>
+              </MenuItem>
+            )}
+          </Menu>
+        </MenuTrigger>
+        <ChevronIcon
+          size={size}
+          className={chevronStyles({direction, isMenu: true})} />
```

Comment:
Noticed that the chevrons become grayed out when the breadcrumbs are disabled, is that supposed to be the case? Looks like they aren't in the designs

Reply:
looks like you are correct